Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -73,18 +78,21 @@ class FCircle
 	float DiameterCircle;
```

Comment:
Remember, it's redundant to call your members stuff like `DiameterCircle` because the fact that it is part of the Circle class is already known. So the name `Diameter` would be enough.

Reply:
changed